sections/introduction/introduction-chapter.tex

```diff
@@ -1,12 +1,12 @@
 \chapter{Recommender systems (RS)}\label{ch:introduction}
-are widely used to combat information overload for users when accessing new information by recommending specific items that the user is likely to find compatible \cite{YouTubeNeural,IndustryPerspective}.
+are widely used to combat information overload for users when accessing new information by recommending specific items that the user is likely to find compatible\cite{YouTubeNeural,IndustryPerspective}.
```

Comment:
Mangler der egentlig ikke noget i starten af den her sætning? linje 2

Reply:
Ikke når du compiler, så kommer chapter titel og det andet til at stå på samme linje... Det er for at der faktisk er et chapter, for det latex "tema" vi bruger er skrald

Reply:
👍